OpenRA.Game/OpenRA.Game.csproj

```diff
@@ -156,6 +156,7 @@
     <Compile Include="Primitives\MergedStream.cs" />
     <Compile Include="Primitives\SpatiallyPartitioned.cs" />
     <Compile Include="Primitives\ConcurrentCache.cs" />
+    <Compile Include="SelectableExts.cs" />
```

Comment:
I think this would be better suited in the `Traits` folder. (especially since it's in the `Traits` namespace)